frontend/lib/src/components/widgets/DateInput/DateInput.test.tsx

```diff
@@ -15,24 +15,28 @@
  */
 
 import React from "react"
-import { screen } from "@testing-library/react"
-import { mount, render } from "@streamlit/lib/src/test_util"
+import "@testing-library/jest-dom"
+import { fireEvent, screen } from "@testing-library/react"
+import { render } from "@streamlit/lib/src/test_util"
 import { WidgetStateManager } from "@streamlit/lib/src/WidgetStateManager"
 import {
   DateInput as DateInputProto,
   LabelVisibilityMessage as LabelVisibilityMessageProto,
 } from "@streamlit/lib/src/proto"
 
-import { Datepicker as UIDatePicker } from "baseui/datepicker"
 import { mockTheme } from "@streamlit/lib/src/mocks/mockTheme"
 import DateInput, { Props } from "./DateInput"
 
+const originalDate = "1970/1/20"
+const fullOriginalDate = "1970/01/20"
+const newDate = "2020/02/06"
+
 const getProps = (elementProps: Partial<DateInputProto> = {}): Props => ({
   element: DateInputProto.create({
     id: "1",
     label: "Label",
-    default: ["1970/01/20"],
-    min: "1970/1/20",
+    default: [originalDate],
```

Comment:
Doesn't seem to matter, but default was previously `fullOriginalDate` and thats what we check for as default in tests below?

Reply:
Ah I can change that but yeah it shouldn't matter technically.